myst_nbs/gaussian_processes/gaussian_process.myst.md

```diff
@@ -6,48 +6,43 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3
+  display_name: Python 3.10.5 ('pymc-dev')
   language: python
   name: python3
 ---
 
+(gaussian_process)=
 # Gaussian Processes using numpy kernel
 
-(c) 2016 by Chris Fonnesbeck
+:::{post} 2016
+:tags: gaussian processes, 
+:category: advanced
+:author: Chris Fonnesbeck
+:::
 
 +++
 
-Example of simple GP fit, adapted from Stan's [example-models repository](https://github.com/stan-dev/example-models/blob/master/misc/gaussian-process/gp-fit.stan).
+Example of simple Gaussian Process fit, adapted from Stan's [example-models repository](https://github.com/stan-dev/example-models/blob/master/misc/gaussian-process/gp-fit.stan).
 
-This example builds a Gaussian process from scratch, to illustrate the underlying model. PyMC3 now includes a dedicated GP submodule which is going to be more usable for a wider variety of problems.
+This example builds a Gaussian process from scratch, to illustrate the underlying model. PyMC now includes a dedicated Gaussian Process submodule which is going to be more usable for a wider variety of problems.
```

Comment:
I would reword this paragraph. I find the "now" particularly problematic, but in general think the idea can be tranmitted more clearly with something like:

```
For illustrative and divulgative purposes, this example builds a Gaussian process from scratch. However, PyMC includes
a {mod}`module dedicated to Gaussian Processes <pymc.gp>` which is recommended instead of coding everything from scratch.
```

Also added the `mod` reference to link to pymc.gp api page.

Reply:
Done!